core/health/rest/src/main/java/org/opennms/core/health/rest/HealthCheckRestService.java

```diff
@@ -47,4 +43,9 @@
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     Response getHealth(@QueryParam("t") @DefaultValue("5000") int timeoutInMs);
+
+    @GET
+    @Path("/tags/{tag}")
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Instead of adding another Rest API, we could add a query param on `getHealth` so that we can supply multiple tags. An example [here](https://github.com/OpenNMS/opennms/blob/4542129338f09206eab222cb0bbd8b0873c19f53/opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/MinionRestService.java#L93)

Reply:
Okay, didn't know we plan to support querying multiple tags. I've updated the code.